skipper.go

```diff
@@ -1647,11 +1650,25 @@ func run(o Options, sig chan os.Signal, idleConnsCH chan struct{}) error {
 		o.CustomFilters = append(o.CustomFilters, compress)
 	}
 
-	// create a filter registry with the available filter specs registered,
-	// and register the custom filters
-	registry := builtin.MakeRegistry()
+	// create a filter registry with the builtin and
+	// custom filter specs registered excluding disabled
+	disabledFilters := make(map[string]struct{})
+	for _, name := range o.DisabledFilters {
+		disabledFilters[name] = struct{}{}
+	}
+
+	registry := make(filters.Registry)
+	registerFilter := func(spec filters.Spec) {
+		if _, disabled := disabledFilters[spec.Name()]; !disabled {
```

Comment:
IMO: disabled -> ok, because this is a very generic code pattern and reads much better than 3x disabled.